### DISCUSSION TITLE
Rust usage inside React Native

### DISCUSSION THREAD

**Discussion by jsoneaday** [reactwg/react-native-releases#16] [Announcements]
I know everyone is busy with the effort to release the new architecture, but as a question looking to the future. Since Meta seems to be a company that uses Rust, I was wondering if longer term there are any plans to switch out the C++ under pinnings of React Native for Rust?

**Comment by cortinico** [comment]
Hey @jsoneaday 
Thanks for your input. Sadly this is the working group for React Native releases. Don't you mind re-opening this discussion in this other repo: github.com/react-native-community/discussions-and-proposals/ ?

**Reply by jsoneaday** [reply]
Oh my apologies. I've moved it to this link https://github.com/react-native-community/discussions-and-proposals/discussions/466

Thank you!
